src/frontend/src/pages/AdminToolsPage/OnboardingConfig/UsefulLinks/UsefulLinksTable.tsx

```diff
@@ -88,7 +89,37 @@ const UsefulLinksTable = () => {
       )}
 
       <Box>
-        <AdminToolTable columns={[{ name: 'Name' }, { name: 'URL' }, { name: '' }]} rows={usefulLinkRows} />
+        <Table>
```

Comment:
What is this added table here?

Reply:
I used MuiTable instead of AdminToolTable because we want it to look a little different from the AdminToolTable and more like the mocks